experiments/composite/pkg/commonoperator/entrypoint.go

```diff
@@ -95,8 +95,11 @@ func (o *Operator) run(ctx context.Context) error {
 	}
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
+		Scheme:             scheme,
+		MetricsBindAddress: metricsAddr,
+		// Metrics: server.Options{
```

Comment:
Just commenting here because I thought this changed (i.e. you need the uncommented code).   I could well be wrong though!

Reply:
https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/experiments/composite/go.mod does not specify a controller-runtime version. Not sure what its picking up. This would only compile with older versions of controller-runtime. Newer versions should cause it to fail to build. Frankly I think the best idea would be to leave it and remove composite/compositions as now dead code.

Reply:
Ah, I missed that, sorry!